src/functions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function displayDreamDetails(array $dreams): string{
+    $dreamOutput ='';
+    foreach(array_reverse($dreams) as $dream) {
```

Comment:
here it's a little unclear why you are reversing the array. i would recommend creating a variable called something like "array in reverse chronological order" or something, which is array_reverse($dreams)

Reply:
it's also worth considering you can achieve that order by modifying your SQL query instead, so i think you should do one of those two options

Reply:
I have done it in the first way